idd/Energy+.idd.in

```diff
@@ -52999,9 +53001,13 @@ AirLoopHVAC:UnitarySystem,
        \type choice
        \key Load
        \key SetPoint
+       \key ASHRAE90VariableFan
```

Comment:
Pausing......I don't really care that much.  I really don't want to make a big discussion, but it seems like the other two keys for that object are specific, clear control terms.  This new key should also be a clear control term.  Is this control mode something specific to only ASHRAE?  Or is it just that ASHRAE requires it?